ledger/participant-integration-api/src/main/scala/platform/config/JwtAudience.scala

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.config
+
+case class JwtAudience(enabled: Boolean, audience: Option[String]) {
+  def targetAudience: Option[String] = if (enabled) {
```

Comment:
The way things are currently implemented this seems to be a global property rather than a property per auth service. Is there a good reason for that? Given that you can specify multiple auth services in Canton that seems a bit confusing. With IDP support you may also want to have different audiences per auth service.